plugins/patcher/src/main/java/org/qbicc/plugin/patcher/ClassPatchInfo.java

```diff
@@ -70,6 +72,10 @@ MethodDeleteInfo getDeletedMethodInfo(final String name, final MethodDescriptor
         return deletedMethods.getOrDefault(name, Map.of()).get(descriptor);
     }
 
+    boolean isDeletedInitializer() {
```

Comment:
Doesn't this also require the lock?  The setter asserts the lock is held so the getter should as well:
```
assert Thread.holdsLock(this);
```

Reply:
Yep, I missed this. Fixed.